code/src/index.css

```diff
@@ -11,3 +11,270 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+body {
+  background-color: rgb(255, 223, 203);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 50px;
+}
+
+h3 {
+  font-weight: 100;
+  font-size: 32px;
+  margin-top: 25px;
+}
+
+h6 {
+  margin-bottom: 0;
+  font-size: 10px;
+}
+
+.todoscreen {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: white;
+  border-radius: 10px;
+  width: 340px;
+  height: 100%;
+  box-shadow: 4px 4px 11px 4px rgb(216, 143, 126);
+  font-weight: 300;
+  padding-top: 10px;
+}
```

Comment:
You would probably see benefits to moving most of these to styled components if you were to continue with this project